edge-modules/MetricsCollector/Program.cs

```diff
@@ -29,18 +32,20 @@ private static async Task<int> MainAsync()
             try
             {
                 moduleClient = await ModuleClient.CreateFromEnvironmentAsync(transportSettings);
+                Dictionary<string, string> testSpecificTags = await GetTestSpecificTagsFromTwin(moduleClient);

```

Comment:
tags may not be test related.  Please rename it.
```suggestion
                Dictionary<string, string> additionalMetricsTags = await GetAdditionalMetricsTagsFromTwin(moduleClient);
```

Reply:
Maybe call log.Info for number of additional tags here